pkgs/applications/video/obs-studio/plugins/obs-tuna/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, stdenv, fetchFromGitHub, obs-studio, cmake, zlib, curl, taglib, dbus
+, pkgconfig, qtbase, wrapQtAppsHook }:
+
+stdenv.mkDerivation rec {
+  pname = "obs-tuna";
+  version = "1.9.6";
+
+  nativeBuildInputs = [ cmake qtbase ];
+  buildInputs = [ obs-studio zlib curl taglib dbus pkgconfig wrapQtAppsHook ];
+
+  src = fetchFromGitHub {
+    owner = "univrsal";
+    repo = "tuna";
+    rev = "v${version}";
+    sha256 = "sha256-+AgRaivvYhogX4CLGK2ylvE8tQoauC/UMvXK6W0Tvog=";
+    fetchSubmodules = true;
+  };
+
+  postInstall = ''
+    mkdir $out/lib $out/share
+    mv $out/obs-plugins/64bit $out/lib/obs-plugins
+    rm -rf $out/obs-plugins
+    mv $out/data $out/share/obs
+  '';
+
+  dontWrapQtApps = true;
+
+  meta = with lib; {
+    description = "Song information plugin for obs-studio";
+    homepage = "https://github.com/univrsal/tuna";
+    license = licenses.gpl2;
+    maintainers = with maintainers; [ shortcord ];
```

Comment:
Please add yourself to the maintainers list: https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix

Reply:
This means there should essentially be two commits in this order:

1. `maintainers: add shortcord`
2. `obs-tuna: init at 1.9.6`